### PR TITLE
refactor/639

### LINKED ISSUE #639: Deprecate `abort` command
The `abort` command is not specific enough to be realistic, nor is it really comprehensive, since only very few actions can actually be "aborted". Similar commands could be created such as `go around / hold position` etc later, though they're hardly needed.

### PR DESCRIPTION
**Pull request by sam-irl**
Resolves #639 

The purpose of this pull request is to deprecate `abort`

Depends on #601 for the docs, I know I updated them here but that's more because I wasn't thinking

### CODE REVIEW

**Comment by sam-irl** [comment]
@n8rzz That really isn't any better, what's the point of just displaying that it *will* be removed, and nothing else? The amount of useful information displayed is the same.

**Comment by n8rzz** [comment]
perhaps.  but at least there is some feedback for the users that are expecting the command to work (and haven't read the documentation)

**Comment by erikquinn** [comment]
I agree with both... Leaving it partially in place only makes the removal process that much more painful. _BUT_ with a command like `abort`, people probably do actually use it (unlike `sr`). So pulling it out from under them may be less tasteful than turning it off and having the game explain to them what happened.

I suppose @n8rzz is right though-- it's better to err on the side of keeping the users informed. We need to save our `we're doing this because it's what's best for the project even if you don't like it` points for later rather than spending them all early and losing the loyalty of some users. :wink:

File: src/assets/scripts/client/commandParser/commandDefinitions.js [hunk 1]
```diff
@@ -71,10 +71,7 @@ const ZERO_ARG_COMMANDS = {
     },
 
     // Aircraft commands
-    abort: {
-        validate: zeroArgumentsValidator,
-        parse: noop
```
Comment (erikquinn) [inline src/assets/scripts/client/commandParser/commandDefinitions.js]: This needs to be present if the command is to register upon a user entering it

File: src/assets/scripts/client/commandParser/commandMap.js [hunk 2]
```diff
@@ -108,7 +108,6 @@ export const COMMAND_MAP = {
     direct: 'direct',
     pd: 'direct',
     dct: 'direct',
-    abort: 'abort',
```
Comment (erikquinn) [inline src/assets/scripts/client/commandParser/commandMap.js]: This needs to be present if the command is to register upon a user entering it

Comment (sam-irl) [reply src/assets/scripts/client/commandParser/commandMap.js]: Hurray, I'm an idiot

Comment (n8rzz) [reply src/assets/scripts/client/commandParser/commandMap.js]: @sam-irl not an idiot at all.  This is why we do pull requests.  Another set (or sets) of eyes is always a good idea!

File: src/assets/scripts/client/aircraft/AircraftCommander.js [hunk 3]
```diff
@@ -216,6 +216,18 @@ export default class AircraftCommander {
...
     /**
+     * Aborts an action. Deprecated.
+     *
+     * @for AircraftCommander
+     * @method runAbort
+     * @param {AircraftModel} aircraft
+     * @return {array} [success of operation, readback]
+     */
+    runAbort(aircraft) {
+        return [false, 'this command has been deprecated, clear an aircraft to another fix or heading instead'];
```
Comment (erikquinn) [inline src/assets/scripts/client/aircraft/AircraftCommander.js]: let's change to `the 'abort' command has been deprecated, please see documentation for help`

**Pull request merged**